### ISSUE TITLE
Missing entrypoint in `v3.2.0`

### ISSUE THREAD

**Issue by F0rce** [storyblok/storyblok-js#620]
### Describe the issue you're facing

> _This is sort of the exact same issue I created over at the `@storyblok/astro` repository (ref https://github.com/storyblok/storyblok-astro/issues/1033)_

When upgrading to `v3.2.0` and running a build command in CI (or any clean environment) the following error message will appear:

```
[commonjs--resolver] Failed to resolve entry for package "@storyblok/js". The package may have incorrect main/module/exports specified in its package.json.
  Stack trace:
    at packageEntryFailure (file:///home/runner/work/<project>/node_modules/.pnpm/vite@6.0.5_@types+node@22.10.2_jiti@2.4.2_yaml@2.6.1/node_modules/vite/dist/node/chunks/dep-BZMjGe_U.js:16462:15)
    at tryNodeResolve (file:///home/runner/work/<project>/node_modules/.pnpm/vite@6.0.5_@types+node@22.10.2_jiti@2.4.2_yaml@2.6.1/node_modules/vite/dist/node/chunks/dep-BZMjGe_U.js:16324:18)
    at Object.handler (file:///home/runner/work/<project>/node_modules/.pnpm/vite@6.0.5_@types+node@22.10.2_jiti@2.4.2_yaml@2.6.1/node_modules/vite/dist/node/chunks/dep-BZMjGe_U.js:51064:15)
    at async PluginDriver.hookFirstAndGetPlugin (file:///home/runner/work/<project>/node_modules/.pnpm/rollup@4.29.1/node_modules/rollup/dist/es/shared/node-entry.js:207[74](https://github.com/<org>/<project>/actions/runs/12486658390/job/34847004770#step:10:75):28)
    at async ModuleLoader.resolveId (file:///home/runner/work/<project>/node_modules/.pnpm/rollup@4.29.1/node_modules/rollup/dist/es/shared/node-entry.js:19807:15)
 ELIFECYCLE  Command failed with exit code 1.
```

### Reproduction

not needed

### Steps to reproduce

1. Upgrade to `v3.2.0`
2. Build
3. Error
4. Downgrade to `v3.1.9`
5. Build
6. Success

### System Info

```shell
System:
    OS: macOS 15.2
    CPU: (8) x64 Intel(R) Core(TM) i7-1068NG7 CPU @ 2.30GHz
    Memory: 550.72 MB / 32.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.10.0 - ~/.local/state/fnm_multishells/44236_1735079062500/bin/node
    npm: 10.2.3 - ~/.local/state/fnm_multishells/44236_1735079062500/bin/npm
    pnpm: 9.15.1 - ~/.local/state/fnm_multishells/44236_1735079062500/bin/pnpm
    bun: 1.1.36 - ~/.bun/bin/bun
  Browsers:
    Chrome: 131.0.6778.205
    Safari: 18.2
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://www.storyblok.com/trust-center#code-of-conduct?utm_source=github.com&utm_medium=readme&utm_campaign=storyblok-js-client)

**Comment by edodusi** [comment]
Should have been solved in #619

**Comment by F0rce** [comment]
Thank you very much, that fixed it :)
